lib/jsdom/browser/htmltodom.js

```diff
@@ -239,17 +239,23 @@ function setChild(core, parentImpl, node) {
   newNode[locationInfo] = node.__location;
 
   if (node.attribs) {
-    for (let localName in node.attribs) {
+    Object.keys(node.attribs).forEach(localName => {
       const value = node.attribs[localName];
-      let prefix = node["x-attribsPrefix"] && node["x-attribsPrefix"][localName] || null;
-      const namespace = node["x-attribsNamespace"] && node["x-attribsNamespace"][localName] || null;
+      let prefix =
+        node["x-attribsPrefix"] &&
+        node["x-attribsPrefix"].hasOwnProperty(localName) &&
```

Comment:
What about `Object.prototype.hasOwnProperty.call(node["x-attribsPrefix"], localName)`? Should I add an additional test for this?